lib/stack.dart

```diff
@@ -33,4 +33,18 @@ class Stack<T> {
 
   /// get the length of the stack.
   int get length => size();
+  
+  // returns true if element is found at the top of the stack
+  bool find(T x) => (x == _list.last) ? true : false;
```

Comment:
@AunSyedShah can you delete this? I don't think the use of `find` in a stack to be useful as we have the `top` method already

Reply:
deleted